src/high-level/constructors.lisp

```diff
@@ -0,0 +1,173 @@
+;;;; constructors.lisp
+;;;;
+;;;; Author: Cole Scott
+
+(in-package #:magicl)
+
+(defconstant +default-tensor-type+ 'double-float)
```

Comment:
I can imagine circumstances in which you may want to set `*default-tensor-type*` to different types (real or complex, etc.) while tinkering in the REPL. If it does not have a negative impact in performance I'd suggest making this a default variable instead of a constant.

Reply:
Changed in c27a0fa